package.json

```diff
@@ -1,7 +1,7 @@
 {
-  "name": "netlify-cli",
+  "name": "ruiramos-netlify-cli",
```

Comment:
Are you sure you mean to push this change to this PR?

Reply:
Apologies @RaeesBhatti. 
This is actually blocking me so I'm trying to publish a version of the cli with this fixed, obviously I screwed up!
The PR should be good now.